style.css

```diff
@@ -383,3 +390,156 @@ hr {
   background-size: cover;
   background-position: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .topnav {
+    height: 39px;
+    background-color: #272a31;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .lowernav {
+    padding: 20px 0;
+  }
+
+  .topnavul {
+    display: flex;
+    gap: 25px;
+    padding-right: 50px;
+  }
+
+  .title h1 {
+    font-size: 44px;
+  }
+
+  .topnavul a {
+    color: #fff;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .navcontainer {
+    order: 2;
+  }
+
+  .navul {
+    display: flex;
+  }
+
+  .navul li {
+    padding-right: 25px;
+  }
+
+  .program a {
+    text-align: center;
+  }
+
+  .mylogo p {
+    font-size: 1.3rem;
+  }
+
+  .navul li a {
+    color: #888;
+    font-size: 19px;
+  }
+
+  .headline {
+    padding: 140px 0;
+  }
+
+  .content {
+    width: 87%;
+    gap: 40px;
+    font-size: 18px;
+  }
+
+  .headline .content p {
+    width: 65%;
+  }
+
+  .headline .content h1 {
+    width: 50%;
+    font-size: 48px;
+    padding-top: 10px;
+  }
+
+  .headline .abouttitle h1 {
+    margin: 10px auto;
+  }
+
+  .abouttitle .title p {
+    display: none;
+  }
+
+  .activities {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+  }
+
+  .activity {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 3fr;
+    align-items: flex-start;
+  }
+
+  .activity li {
+    text-align: center;
+    font-size: 15px;
+    line-height: 30px;
+  }
+
+  .activity .title {
+    font-size: 19px;
+  }
+
+  .art {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .artist .about .desc {
+    line-height: 24px;
+    font-size: 13px;
+  }
+
+  .partiner .content {
+    width: 85%;
+  }
+
+  .partiners img {
+    width: 110px;
+  }
```

Comment:
- [ ] Kindly fix the design for the partner section to match it like the design in Behance and also consider fixing the design for the footer to match the design.
> You can use the following code to fix the design for the partners' section.
```CSS
.partiners {
    display: flex;
    padding: 10px 0 30px 0;
    justify-content: center;
    gap: 10%;
    font-size: 64px;
    color: #82808a;
}
```
| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/190661496-bcd2565f-c0ee-4529-9c5d-308dbfbdf47f.png) | ![image](https://user-images.githubusercontent.com/53402600/190661570-85326da9-e03f-4424-a84b-a77a7aabf00e.png) |